runtime/arm64.S

```diff
@@ -333,8 +333,6 @@ FUNCTION(caml_start_program)
     /* Return to C caller */
         ret
         CFI_ENDPROC
-        .type   .Lcaml_retaddr, %function
-        .size   .Lcaml_retaddr, .-.Lcaml_retaddr
```

Comment:
It could be that these declarations on local labels are a leftover from the ARM (32 bits) port, where I think they are needed for correct ARM/Thumb interoperability.  It could also be that these declarations serve a purpose in the ARM64/Linux port.  So please leave them and wrap them if `#ifndef macos` later.

Reply:
I tried that, the shasum of the runtime is exactly the same after removing it on Linux ARM64, I tested commit to commit here on a Raspberry Pi 4 running AArch64 to ensure that it was working without any regressions. And some of them related to the runtime behavior I checked the shasum of the generated arm64.o

Reply:
I confurm those directives on `.Lcaml_retaddr` and `.Ltrap_handler` seem to make no difference on the compiled `arm64.o` under Linux.  So, commit 7654d56 can stay.